openvdb_points/tools/AttributeArray.h

```diff
@@ -233,6 +235,14 @@ class AttributeArray
     /// Return @c true if this attribute is not serialized during stream output.
     bool isTransient() const { return bool(mFlags & TRANSIENT); }
 
+    /// @brief Specify whether this attribute is for tracking group membership
+    /// @note  Attributes are not group attributes by default.
+    void setGroup(bool state);
+    /// Return @c true if this attribute is for tracking groups
+    bool isGroup() const { return bool(mFlags & GROUP); }
+
+    uint16_t flags() const { return mFlags; }
```

Comment:
Would be useful to add a doc comment here, perhaps with a reference to the Flag enum